README.md

```diff
@@ -58,55 +58,152 @@ documentation](https://www.packer.io/docs).
-   Open the `.cirrus.yml` file, and paste the **big number** *prefixed by the
-   letter 'c'*, in place of the value next to `_BUILT_IMAGE_SUFFIX:`.  For
-   example, if the url was `http://.../12345` you would paste in `c12345`
-   as the value for `_BUILT_IMAGE_SUFFIX:`.
+5. Go over to whatever other containers/repository needed the image update.
+   Open the `.cirrus.yml` file, and find the 'env' line referencing the image
+   suffix.  It will likely be named `_BUILT_IMAGE_SUFFIX:` or something similar.
 
-7. Open up a PR with this change, and push it.  Once all tests pass and your
+7. Paste in the **big number** *prefixed by the letter 'c'*.  The *"c*" indicates
+   the images are *cache images*.  For example, if the url was `http://.../12345`
+   you would paste in `c12345` as the value for `_BUILT_IMAGE_SUFFIX:`.
+
+8. Open up a PR with this change, and push it.  Once all tests pass and you're
    satisfied with the image changes, ask somebody to review/approve both
    PRs for merging.  If you're feeling generous, perhaps provide cross-links
    between the two PRs in comments, for future reference.
 
-8. After all the PRs are merged, you're done.  You may now attend to the little
-   dog begging you for a walk for the last hour.  Hurry!  Little-dogs, do not
-   have big-dog bladders!
+9. After all the PRs are merged, you're done.
+
+
+
+
+## The image-builder image (overview step 1)
+
+Google compute engine (GCE) does not provide a wide selection of ready-made
+VM images for use.  Instead, a lengthy and sophisticated process is involved
+to prepare, properly format, and import external VM images for use.  In order
+to perform these steps within automation, a dedicated VM image is needed which
+itself has been prepared with the necessary incantations, packages, configuration,
+and magic license keys.
+
+For normal day-to-day use, this process should not need to be modified or
+maintained much.  However, on the off-chance that's ever not true, here is
+an overview of the process followed **by automation** to produce the
+*image-building VM image*:
+
+1. Build the container defined in the `ci` subdirectory's `ContainerFile`.
+   Start this container with a copy of the current repository code provided
+   in the `$CIRRUS_WORKING_DIR` directory. For example on a Fedora host:
+
+   ```
+   podman run -it --rm --security-opt label=disable -v $PWD:$PWD -w $PWD ci_image
+   ```
+
+2. From within the `ci` container (above), in the repository root volume, execute
+   the  `make image_builder` target.
+
+3. The `image_builder/gce.yml` file is converted into JSON format for
+   consumption by the [Hashicorp *packer* utility](https://www.packer.io/).
+   This generated file may be ignored, *make* will be regenerate it upon
+   any changes to the YAML file.
+
+4. Packer will spin up a GCE VM based on CentOS, installs necessary packages
+   and attaches a [nested-virtualization "license" to the
```

Comment:
The change here now leaves "installs" and "attaches" disagreeing with "will spin up"; but this isn't worth re-pushing for

Reply:
Nope, I got it.